### PR TITLE
Update change-log.md

### PR DESCRIPTION
**Pull request by MartaD**

### CODE REVIEW

File: docs/change-log.md [hunk 1]
```diff
@@ -2,13 +2,25 @@
 
 This document provides an overview of the changes we are making over time. Feel free to reach out to us in case of any specific questions.
 
+## Product Update August 4th, 2023
+
+- **Improved:** A large number of [Container Image](https://developer.humanitec.com/platform-orchestrator/working-with/workloads/manage-containers/#create-a-container) versions no longer impairs the performance of the version selection component.
```
[the remark this answers is not in the document]
Comment (MartaD) [reply docs/change-log.md:7]: we're preventing a huge degradation, if it's too many versions they all were loaded in the UI anyway and it became unresponsive

[the remark this answers is not in the document]
Comment (sujaya-sys) [reply docs/change-log.md:7]: ok! 👍

File: docs/change-log.md [hunk 2]
```diff
@@ -2,13 +2,25 @@
...
+  
+- **Fixed:** [CPU settings of a workload](https://developer.humanitec.com/platform-orchestrator/reference/workload-profiles/features/#container-resources) are now always visible in the UI
```
[the remark this answers is not in the document]
Comment (MartaD) [reply docs/change-log.md:9]: that was my understanding, here's the source ticket: https://humanitec.atlassian.net/browse/WAL-6993

File: docs/change-log.md [hunk 3]
```diff
@@ -2,13 +2,25 @@
...
+  
+- **Fixed:** Issue where DNS name was not visible in a workload opened with a direct URL.
+  
+- **Fixed:** Issue where Git user name could not be reset to a blank string in a [terraform](https://developer.humanitec.com/drivers/built-in-drivers/generic-terraform/) resource definition.
```
[the remark this answers is not in the document]
Comment (MartaD) [reply docs/change-log.md:13]:
```suggestion
- **Fixed:** Inputs can now be reset to a blank string in the [terraform driver](https://developer.humanitec.com/drivers/built-in-drivers/generic-terraform/) based resource definition. 
```

how about this one?

File: docs/change-log.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (sujaya-sys) [reply docs/change-log.md:9]: Ah, this was a Score only issue I believe

[the remark this answers is not in the document]
Comment (sujaya-sys) [reply docs/change-log.md:9]:
```suggestion
- **Fixed:** The [CPU settings of a workload](https://developer.humanitec.com/platform-orchestrator/reference/workload-profiles/features/#container-resources) are now visible in the UI when creating a delta via the [score-humanitec CLI. ](https://github.com/score-spec/score-humanitec).
```

File: docs/change-log.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (MartaD) [reply docs/change-log.md:7]: Here's the request: https://humanitec-support.slack.com/archives/G0159AYS7L1/p1682319601748939

File: docs/change-log.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (sujaya-sys) [reply docs/change-log.md:13]: Sure, works too. Would just remove `the` i.e. `..blank string in [terraform driver](https://developer.humanitec.com/drivers/built-in-drivers/generic-terraform/) based resource definitions.`

File: docs/change-log.md [hunk 4]
```diff
@@ -2,13 +2,25 @@
 
 This document provides an overview of the changes we are making over time. Feel free to reach out to us in case of any specific questions.
 
+## Product Update August 4th, 2023
+
+- **Improved:** A large number of [Container Image](https://developer.humanitec.com/platform-orchestrator/working-with/workloads/manage-containers/#create-a-container) versions no longer impairs the performance of the version selection component.
+  
+- **Fixed:** The [CPU settings of a workload](https://developer.humanitec.com/platform-orchestrator/reference/workload-profiles/features/#container-resources) are no longer hidden in the UI when creating a delta via the [score-humanitec CLI. ](https://github.com/score-spec/score-humanitec).
```
Comment (sujaya-sys) [inline docs/change-log.md:9]:
```suggestion
- **Fixed:** The [CPU settings of a workload](https://developer.humanitec.com/platform-orchestrator/reference/workload-profiles/features/#container-resources) are no longer hidden in the UI when creating a delta via the [score-humanitec CLI](https://github.com/score-spec/score-humanitec).
```

**Review by sujaya-sys — APPROVED** [review]

**Pull request merged**